pyrsp/utils.py

```diff
@@ -31,6 +31,24 @@ def unpack(pkt):
     pkt = pkt[1:-3]
     return pkt
 
+def decode_data(data):
+    """ decodes data from received packet, decoding algorithm is described
+here: https://sourceware.org/gdb/onlinedocs/gdb/Overview.html
+    """
+    if data.find('*') == -1:
+        return data
```

Comment:
oooh. i overlooked that. does that mean "*" cannot occur in data only when it's rle encoded?

Reply:
Are you mean: "does that mean "\*" ~~cannot~~ occur in data only when it's rle encoded?"
If that's true, then my answer is Yes. Because "*" is only occur in encoded data.